modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_analytics-clould-modal.scss

```diff
@@ -0,0 +1,4 @@
+.cp-analytics-cloud-status-modal {
+	height: 354px !important;
+	width: 600px !important;
```

Comment:
Theses `!important` are required?
As a good practice, should be good avoid use them whenever possible :)

https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/css/when_to_use_css_utility_classes.md#2-the-important-attribute

Reply:
Thank's!